.github/workflows/gradle.yml

```diff
@@ -30,7 +31,9 @@ jobs:
       run: cd cadc-datalink-server && ../gradlew --info clean build javadoc checkstyleMain install
       
     - name: build and test cadc-soda-server
-      run: cd cadc-soda-server && ../gradlew --info clean build javadoc checkstyleMain install
-      
+
+    - name: install WCSLIB (libwcs7)
```

Comment:
The published cadc-wcs only supports wcslib5; I came up with a scheme to include JNI binding lib for multiple wcslib versions, but the grunt work to build and test for wcslib6 and 7 hasn't been done. Will create an issue for that work.